src/Glob.jl

```diff
@@ -202,6 +204,8 @@ function _match(pat::String, i0, c::Char, caseless::Bool, extended::Bool) # retu
     mc, j = next(pat, i)
     negate = false
     if mc == '!'
+        h
+
```

Comment:
Hmmm that's from the first commit, looks like a typo.

I guess this means this case is not covered, any ideas how to hit it?